ompi/Makefile.am

```diff
@@ -128,9 +129,9 @@ DIST_SUBDIRS = \
 
 # Build the main MPI library
 
-lib_LTLIBRARIES = libmpi.la
-libmpi_la_SOURCES =
-libmpi_la_LIBADD = \
+lib_LTLIBRARIES = lib@OMPI_LIB_PREFIX@.la
```

Comment:
No objection to doing the Fortran names also.  I think longer term the consistency would be desirable.  Short term (for our 2.0 based release), we're only interested in the libmpi_suffix, and if the community default of empty string is applied, this change should be an effective no-op for the Open MPI v2.x branch.